exercises/robot-simulator/src/test/java/RobotTest.java

```diff
@@ -15,11 +15,11 @@ public void testRobotIsCreatedWithCorrectInitialPosition() {
 
     @Ignore("Remove to run test")
     @Test
-    public void testRobotIsCreatedWithCorrectInitialOrientation() {
-        final Orientation initialOrientation = Orientation.NORTH;
-        final Robot robot = new Robot(new GridPosition(0, 0), initialOrientation);
+    public void testRobotIsCreatedWithCorrectInitialPosition() {
```

Comment:
The build is failing because there already is a test with this name. From what I can see from the test data this should be called something like `testNegativePositionsAreAllowed`? 🙂